File: ClientApp/tmetric-statistics-app/src/domain/CalendarWeekData.ts
export class CalendarWeekData {
  public calendarWeek: number;
  public startOfCalendarWeek: string;
  public endOfCalendarWeek: string;
  public plannedHours: number;
  public actualHours: number;
  public overtime: number;
  public sickLeave: number;
  public holidays: number;
  public bridgeDay: number;
  public publicHoliday: number;

  constructor(
    calendarWeek?: number,
    startOfCalendarWeek?: string,
    endOfCalendarWeek?: string,
    plannedHours?: number,
    actualHours?: number,
    overtime?: number,
    sickLeave?: number,
    holidays?: number,
    bridgeDay?: number,
    publicHoliday?: number

  ) {
    this.calendarWeek = calendarWeek;
    this.startOfCalendarWeek = startOfCalendarWeek;
    this.endOfCalendarWeek = endOfCalendarWeek;
    this.plannedHours = plannedHours;
    this.actualHours = actualHours;
    this.overtime = overtime;
    this.sickLeave = sickLeave;
    this.holidays = holidays;
    this.bridgeDay = bridgeDay;
    this.publicHoliday = publicHoliday;
  }

}

File: Model/CalendarWeekData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace tmetricstatistics.Model
{
    public class CalendarWeekData
    {
        public int calendarWeek { get; set; }
        public String startOfCalendarWeek { get; set; }
        public String endOfCalendarWeek { get; set; }
        public double plannedHours { get; set; }
        public double actualHours { get; set; }
        public double overtime { get; set; }
        public int sickLeave { get; set; }
        public int holidays { get; set; }
        public int bridgeDay { get; set; }
        public int publicHoliday { get; set; }

    }
}
